tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -1909,6 +1909,100 @@ TEST (testTensorTransform, arithmeticChangeOptionString)
   gst_harness_teardown (h);
 }
 
+/**
+ * @brief Test for tensor_transform arithmetic (flex tensor)
```

Comment:
I think static tensor -> flex tensor transform test is needed to increase the code coverage.

Reply:
later work : transform-caps in tensor-transform
When called transform-caps(), it should append all possible caps as result, but now only one possible caps supported.